build.sh

```diff
@@ -2,11 +2,30 @@
 
 BASE=$(cd $(dirname $0) && pwd)
 
-VERSION=`cat pom.xml | grep '<version>' | head -1 | tail -1 | sed 's_.*<version>\([^<]*\)</version>.*_\1_'`
+VERSION=`cat pom.xml | grep -A1 '<artifactId>druid</artifactId>' | grep '<version>' | head -1 | sed 's_.*<version>\([^<]*\)</version>.*_\1_'`
```

Comment:
@housejester,  @xvrl actually removed the build script because it was unnecessary and not very useful. It is easier to just use mvn package.